client/src/styles/WorkflowField.scss

```diff
@@ -57,8 +57,8 @@
           background: -o-linear-gradient($white, $grey-lightish);
           background: -ms-linear-gradient($white, $grey-lightish);
           background: linear-gradient($white, $grey-lightish);
-          -webkit-text-shadow: $white 0 1px 1px;
-          -moz-text-shadow: $white 0 1px 1px;
+          -webkit-text-shadow: $white 0 1px 1px; // sass-lint:disable-line no-misspelled-properties
+          -moz-text-shadow: $white 0 1px 1px; // sass-lint:disable-line no-misspelled-properties
```

Comment:
This is properly supported by all browsers. I don't think we should keep the old CSS properties.

```suggestion
```